webplugin/template/chat.html

```diff
@@ -48,8 +49,9 @@
                 if (skipRouting){
                     defaultSettings["skipRouting"] = !!skipRouting;
                 }
-                
+
                 var kommunicateSettings = {
+                    "inputFocus" : isInputFocussed == "true",
```

Comment:
use better and more descriptive name @HrithikKrishna56

Reply:
Changed